modules/nf-core/modules/rtsstat/main.nf

```diff
@@ -0,0 +1,36 @@
+// Import generic module functions
```

Comment:
Please move to `modules/local`

Reply:
I think all modules have been shifted modules/local